<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2614</width>
    <height>1103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RF GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_genray_filepath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filepath</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eqdsk_filepath">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_powerscale">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Powerscale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_powerscale">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_1st_harm">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1st Damping Harmonic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1st_harm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_harms">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of Harmonics</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_harms">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ion_Species_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ion Species 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>2390</x>
      <y>970</y>
      <width>168</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_data_quit">
    <property name="geometry">
     <rect>
      <x>2157</x>
      <y>970</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Save Data and Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_data">
    <property name="geometry">
     <rect>
      <x>1980</x>
      <y>970</y>
      <width>168</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&amp;Save Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eqdsk_filepath_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>90</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_powerscale_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Powerscale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1st_harm_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>190</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1st Damping Harmonic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1st_harm_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>190</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_harms_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>240</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ion_Species_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ion Species 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_harms_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>240</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of Harmonics</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_powerscale_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>140</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_genray_filepath_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filepath</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_eqdsk_filepath_3">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>90</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_powerscale_3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>140</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Powerscale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1st_harm_3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>190</y>
      <width>251</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1st Damping Harmonic</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1st_harm_3">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>190</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_harms_3">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>240</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ion_Species_3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>30</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ion Species 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_harms_3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>240</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of Harmonics</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_powerscale_3">
    <property name="geometry">
     <rect>
      <x>1860</x>
      <y>140</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_genray_filepath_3">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>90</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filepath</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
